ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -3110,3 +3110,8 @@ literals:
 - ultraviolet_irradiance_sensor:
     fixed_min: 0.0
     flexible_max: 2000.0
+
+- heater_run_mode:
```

Comment:
Can we instead use two heater run commands? I.e. `heater_run_command_1` and `heater_run_command_2`.